scripts/nolints.json

```diff
@@ -2160,6 +2157,8 @@
  ["docBlame", "Lean.Meta.Simp.mkCast"],
  ["docBlame", "Lean.Meta.Simp.mkEqSymm"],
  ["docBlame", "Lean.Meta.findCore.check"],
+ ["docBlame", "Lean.Meta.mkRichHCongr.doubleTelescope.loop"],
+ ["docBlame", "Lean.Meta.mkRichHCongr.withNewEqs.loop"],
```

Comment:
How about we just write the doc-strings, even if they are just `implementation of ...`?

Reply:
Hopefully one day Lean will tag local declarations that come from things like `let rec` to help the linter...